src/options/components/navigation/NavLink.jsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Link } from 'react-router'
+import classNames from 'classnames'
+import styles from './styles.css'
+
+const NavLink = ({ route, currentLocation, state }) => {
+    const navClasses = classNames({
+        [styles.navLink]: true,
+        [styles.isActive]: isActive(route),
+    })
+
+    const navIcon = classNames({
+        [styles.navIcon]: true,
+        'material-icons': true,
+    })
+
+    function isActive(route) {
+        return currentLocation.pathname === route.pathname
+    }
+
+    return (
+        <li>
+            <div className={navClasses}>
+                <i className={navIcon}>{route.icon}</i>
+                {route.component === 'faq' && (
+                    <a
+                        className={navClasses}
+                        href={route.pathname}
+                        target="_blank"
+                    >
+                        {route.name}
+                    </a>
+                )}
+                {route.component !== 'faq' && (
+                    <Link className={navClasses} to={route.pathname}>
+                        {route.name}
+                    </Link>
+                )}
+            </div>
+            {route.name === 'Import' &&
+                isActive(route) && (
+                    <div className={styles.importSubItems}>
+                        <div
+                            className={
```

Comment:
use the `classnames` package to make conditionally applying styles classes less messy, like this. There's other example in these view components of how you can use this package